TicTacToe.c

```diff
@@ -0,0 +1,126 @@
+﻿// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// Ayisha Sameera,GET.
+// ------------------------------------------------------------------
+// TicTacToe.c
+// Program on branch Test3.1
+// Program implements TiTacToe with 2 or 1 player mode.
+// Program displays the board upon valid moves.
+// ------------------------------------------------------------------------------------------------
+#include <stdio.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <io.h>
+#include <fcntl.h>
+#include <string.h>
+
+#define DIA1 6
+#define DIA2 7
+
+static int TicTacToe (char* grid, int index, int Player) {
+   // 3 x 3 Grid consists of three rows, three columns and two diagonals(8 possibilities)
+   static int Count[8][2] = {

```

Comment:
static variables start with s.
```suggestion
   static int sCount[8][2] = {

```